package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-edge-devtools",
-    "version": "1.0.4",
+    "version": "1.0.6",

```

Comment:
Shouldn't this be in the package.json file?

Reply:
it is 😃 , this PR bumped the package.json version:
https://github.com/microsoft/vscode-edge-devtools/commit/f7ab8a7a9d5b14a5732c631f4ab8f290ea154e00

Reply:
Oh i see. So it was actually my fault oops!